sdk/python/cmd/pulumi-language-python/main_test.go

```diff
@@ -234,3 +254,24 @@ func TestDeterminePulumiPackages(t *testing.T) {
 		assert.NotEmpty(t, old.Location)
 	})
 }
+
+func runPythonCommand(t *testing.T, virtualenv, cwd string, arg ...string) ([]byte, error) {
+	t.Helper()
+
+	tc, err := toolchain.ResolveToolchain(toolchain.PythonOptions{
+		Toolchain:  toolchain.Pip,
+		Root:       cwd,
+		Virtualenv: virtualenv,
+	})
+	require.NoError(t, err)
+	cmd, err := tc.Command(context.Background(), arg...)
```

Comment:
We used to pass `ctx` into `runPythonCommand`.  Why did we change that and use `context.Background()` here instead?

Reply:
This is a test helper only, and all the uses of it pass in `context.Background()`.